server/lib/renderers/html.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @description
+ * This service is responsible for compiling valid HTML documents given a path to a handlebars template
+ * 
+ * @requires express-handlebars
+ * @requires server/lib/template
+ */
+const exhbs = require('express-handlebars');
```

Comment:
If you put 'use strict' above this line with `const`, Travis will pass.

For Node versions 5 and under, `let`, `const`, and others have to be within 'strict' mode.  Node 6 removes this restriction.

Unfortunately, our error stack traces only went as far as `invoice.js` which requires this file.  Perhaps we should look into handlers that give us long stack traces.